src/software/simulation/simulator_robot.h

```diff
@@ -278,10 +278,12 @@ class SimulatorRobot
     typedef struct DribblerBall_t
     {
         PhysicsBall* ball;
-        // We keep track of whether or not the ball can be kicked.
-        // This extra information is used to prevent edge cases like
-        // the ball getting kicked/chipped multiple times.
-        bool can_be_chicked;
+        // We keep track of whether or not the ball can be controlled
+        // in any way by the robot. This extra information is used to
```

Comment:
Would appreciate clarification on what controlled means, i.e. can be kicked, chipped, or dribbled

Reply:
clarified in the comment to include chipping kicking and dribbling